### PR TITLE
Add Livida Blog Support

### PR DESCRIPTION
**Pull request by MegaJoshy**
Examples:
image
image
image

### CODE REVIEW

File: Livida Blog/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "author": {
+      "name": "Livida",
+      "id": "559160676485169161"
```
Comment (Bas950) [inline Livida Blog/dist/metadata.json]: is this your discord name and id?

Comment (MegaJoshy) [reply Livida Blog/dist/metadata.json]: It's the account for my community but I can change it to my own if needed.

Comment (Bas950) [reply Livida Blog/dist/metadata.json]: usually its your own account, since it would be used if the presence is broken for contacting the dev.

Comment (MegaJoshy) [reply Livida Blog/dist/metadata.json]: Ah alright, I'll change to my own then.

**Comment by Bas950** [comment]
Best First-time contributor I have seen in a while, all looks good

**Pull request merged**